### PR TITLE
1 1 1 release notes

### PR DESCRIPTION
**Pull request by duderino**

### CODE REVIEW

File: content/about/notes/1.1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-25
+icon: notes
+---
+
+Istio 1.1.1 is the first in a series of patch releases for Istio 1.1, which we will create at least weekly assuming we have changes to share.
+
+This release includes security vulnerability fixes and improvements to robustness.
+
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: No blank line.

File: content/about/notes/1.1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-25
+icon: notes
+---
+
+Istio 1.1.1 is the first in a series of patch releases for Istio 1.1, which we will create at least weekly assuming we have changes to share.
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: I'd still remove this paragraph. It's a patch release like any other patches, and we will produce them when we need to.

**Comment by geeknoid** [comment]
Shoot, I forgot one more change that's needed with this. Plesse update the full_version field in the data/args.yml file. This will update the version info in a few places on the site, including the download button on the home page.

File: content/about/notes/1.1.1/index.md (hunk 1 above)
Comment (duderino) [reply content/about/notes/1.1.1/index.md]: done

File: content/about/notes/1.1.1/index.md (hunk 2 above)
Comment (duderino) [reply content/about/notes/1.1.1/index.md]: done

**Comment by duderino** [comment]
> Shoot, I forgot one more change that's needed with this. Plesse update the full_version field in the data/args.yml file. This will update the version info in a few places on the site, including the download button on the home page.

ACK

**Pull request merged**